posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: "Meet our open-source champions: Billy Dawson"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/BillyD73
+author_github: https://github.com/BillyD73
+seo-title: Meet our open-source champions: Billy Dawson
+seo-description: DESCRIPTION
+blog_description: "DESCRIPTION"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
```

Comment:
replace with an image from the post in the form

https://openliberty.io/img/blog/image-file.png